app/code/Bulbulatory/Recommendations/Model/RecommendationRepository.php

```diff
@@ -2,14 +2,19 @@
 
 namespace Bulbulatory\Recommendations\Model;
 
+use Exception;
+use Throwable;
 use Magento\Framework\Api\SearchCriteriaInterface;
 use Magento\Framework\Exception\NoSuchEntityException;
 use Bulbulatory\Recommendations\Api\RecommendationRepositoryInterface;
 use Bulbulatory\Recommendations\Model\ResourceModel\Recommendation\CollectionFactory as RecommendationCollectionFactory;
-use Bulbulatory\Recommendations\Model\Recommendation;
+use Magento\Framework\Math\Random;
 
 class RecommendationRepository implements RecommendationRepositoryInterface
 {
+    const STATE_UNCONFIRMED = 0;
+    const STATE_CONFIRMED   = 1;

```

Comment:
States are related to a single recommendation so I think a better place for these consts would be the Recommendation model.

Reply:
@TM18

> States are related to a single recommendation so I think a better place for these consts would be the Recommendation model.

I change const place to **Bulbulatory\Recommendations\Model\Recommendation** class